app/models/miner_ranking.rb

```diff
@@ -26,7 +26,7 @@ def ranking_infos
     addresses = Address.where(id: address_ids)
     ranking_infos = []
     addresses.find_each do |address|
-      block_ids = address.ckb_transactions.available.where(is_cellbase: true).select("block_id")
+      block_ids = address.ckb_transactions.where(is_cellbase: true).select("block_id")
```

Comment:
```suggestion
      block_ids = address.ckb_transactions.where(is_cellbase: true).pluck("block_id")
```

Reply:
`select` will return relation. `pluck` will return array. if `select` reduce the amount of data the query？